python/lsst/ap/association/packageAlerts.py

```diff
@@ -405,6 +409,15 @@ def produceAlerts(self, alerts, visit, detector):
                         f.write(alertBytes)
 
         self.producer.flush()
+        visit_midpoint = Time(alert["diaSource"]["midpointMjdTai"], format='mjd', scale='utc')
```

Comment:
Shouldn't this be `scale='tai'`? That would mean a difference of 37s, if I recall correctly.
Edit: see below, we should get this time directly from the exposure and not from the source catalog (which derives the time from the exposure).